features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/RumConfigurationBuilderTest.kt

```diff
@@ -78,39 +81,43 @@ internal class RumConfigurationBuilderTest {
         val rumConfiguration = testedBuilder.build()
 
         // Then
-        assertThat(rumConfiguration.featureConfiguration).isEqualTo(
-            RumFeature.Configuration(
-                customEndpointUrl = null,
-                sampleRate = RumFeature.DEFAULT_SAMPLE_RATE,
-                telemetrySampleRate = RumFeature.DEFAULT_TELEMETRY_SAMPLE_RATE,
-                telemetryConfigurationSampleRate = RumFeature.DEFAULT_TELEMETRY_CONFIGURATION_SAMPLE_RATE,
-                userActionTracking = true,
-                touchTargetExtraAttributesProviders = emptyList(),
-                interactionPredicate = NoOpInteractionPredicate(),
-                viewTrackingStrategy = ActivityViewTrackingStrategy(false),
-                viewEventMapper = NoOpEventMapper(),
-                errorEventMapper = NoOpEventMapper(),
-                actionEventMapper = NoOpEventMapper(),
-                resourceEventMapper = NoOpEventMapper(),
-                longTaskEventMapper = NoOpEventMapper(),
-                telemetryConfigurationMapper = NoOpEventMapper(),
-                longTaskTrackingStrategy = MainLooperLongTaskStrategy(100L),
-                backgroundEventTracking = false,
-                trackFrustrations = true,
-                // on Android R+ this should be false, but since default value is static property
-                // RumFeature.DEFAULT_RUM_CONFIG, it is evaluated at the static() block during class
-                // loading, so we are not able to set Build API version at this point. We will test
-                // it through a helper method in RumFeature.Companion
-                trackNonFatalAnrs = true,
-                vitalsMonitorUpdateFrequency = VitalsUpdateFrequency.AVERAGE,
-                sessionListener = NoOpRumSessionListener(),
-                additionalConfig = emptyMap(),
-                initialResourceIdentifier = TimeBasedInitialResourceIdentifier(),
-                lastInteractionIdentifier = TimeBasedInteractionIdentifier(),
-                trackAnonymousUser = true,
-                slowFramesConfiguration = null
-            )
-        )
+        rumConfiguration.featureConfiguration.apply {
```

Comment:
minor: maybe `with(rumConfiguration.featureConfiguration)` it will be more descriptive? like to signify that we are not going to mutate it.